core/domain/stats_services.py

```diff
@@ -340,88 +279,78 @@ def update_exp_issues_for_new_exp_version(
             exploration.id, exploration.version - 1)
         return
 
-    # Handling reverts.
     if revert_to_version:
         old_exp_issues = get_exp_issues(exploration.id, revert_to_version)
-        # If the old exploration issues model doesn't exist, the current model
-        # is carried over (this is a fallback case for some tests, and can
-        # never happen in production.)
```

Comment:
This comment isn't true, and provides negative value. I've decided to remove it because of this.

Reply:
Could you please give an explanation of how?

Reply:
I've fixed the tests this comment is describing, so we can consider this comment obsolete.